packages/volto/src/components/theme/Logo/Logo.jsx

```diff
@@ -34,26 +46,20 @@ const Logo = () => {
   }, [dispatch, pathname]);
 
   const navRootPath = flattenToAppURL(navroot?.navroot?.['@id']) || '/';
-  const currentURLIsNavRoot = pathname !== navRootPath;
 
   return (
-    <ConditionalLink
-      href={navRootPath}
-      title={navroot?.navroot?.title}
-      // In case that the content returns 404, there is no information about the portal
-      // then render the link anyways to get out of the Unauthorized page
-      condition={!navroot || currentURLIsNavRoot}
-    >
+    <a href={navRootPath} aria-label={intl.formatMessage(messages.home)}>
```

Comment:
@Molochem I just realised of this. Is there any rationale behind it is not a react router link but a plain link?

Reply:
https://github.com/plone/volto/pull/5981

Reply:
Nope , just an oversight by me. Thx for fixing.